### ISSUE TITLE
Question: Make NetVips work with graphicsmagick

### ISSUE THREAD

**Issue by avaver** [kleisauke/net-vips#223]
Hi, I have spent 3 days already trying to make NetVips to work with bmp files in linux docker container with no luck so far. 
Initially I have been using NetVips with pre-compiled linux-x64 binaries.
Then I realized I need libvips library with magick support, so I have compiled it with `-Dmagick=enabled -Dmagick-package=GraphicsMagick`
I used this script https://github.com/kleisauke/libvips-packaging/blob/main/build/lin.sh with my modifications:
```
mkdir ${DEPS}/gm
$CURL https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/${VERSION_GRAPHICSMAGICK}/GraphicsMagick-${VERSION_GRAPHICSMAGICK}.tar.xz/download | tar xJC ${DEPS}/gm --strip-components=1
cd ${DEPS}/gm
./configure --host=${CHOST} --prefix=${TARGET} --enable-static --disable-shared --disable-dependency-tracking 
make install-strip
```

also I had to add ` -fopenmp` compiler option

It compiled succesfully, and my test C# program was working fine with newly compiled library when run on Ubuntu.

But when I try to use my compiled library in Debian-based docker container, it fails with `VipsForeignLoad: buffer is not in a known format`
I tried to install different packages in container to make it work (like imagemagick, graphicsmagick, even libvips-dev) and I got it working at some point but I didn't note after what step it started working, and now I cannot get it to work again...

So basically my question is what do I need to do to enable BMP support and use it from NetVips. Thank you!

**Comment by kleisauke** [comment]
Modifying the scripts in the https://github.com/kleisauke/libvips-packaging repo is not straightforward and will require experience with shell scripting and compiling Linux software from source. It would be much easier if you install libvips via your package manager (see e.g. https://github.com/libvips/libvips/issues/1184).

I usually recommend using Rocky Linux (or any other RHEL derivative) in combination with Remi's RPM repository, a third-party repository that includes an up-to-date version of libvips. I've made an example with BMP support via GraphicsMagick here:
https://gist.github.com/kleisauke/cc932ca1ff64f4442f69f8b5384388fb
```console
$ git clone https://gist.github.com/cc932ca1ff64f4442f69f8b5384388fb.git netvips-dockerized
$ cd netvips-dockerized/
$ docker build -t netvips-docker .
$ docker run -it --rm netvips-docker
libvips 8.15.1
<NetVips.Image 1419x1001 Uchar, 3 bands, Srgb>
```

**Comment by kleisauke** [comment]
I hope this information helped. Please feel free to re-open with more details if further assistance is required.
